docs/source/install/path.rst

```diff
@@ -114,9 +114,16 @@ References
 .. [Spack]
         T. Gamblin and contributors.
         *A flexible package manager that supports multiple versions, configurations, platforms, and compilers*,
+        SC '15 Proceedings of the International Conference for High Performance Computing, Networking, Storage and Analysis (2015),
```

Comment:
OPTIONAL:
On the [web site](https://www.ieee.org/conferences_events/conferences/conferencedetails/index.html?Conf_ID=32761) it says: `SC15: International Conference for High Performance Computing, Networking, Storage and Analysis` so no `SC '15` but a `SC15` instead.